src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -175,7 +175,6 @@ namespace AppInstaller::CLI::Execution
             std::string response;
             if (!std::getline(m_in, response))
             {
-                m_in.get();

```

Comment:
?

Reply:
I somehow find through commit history and saw it's incorrectly added during a merge. There's a throw right after. So I removed it.